language/move-lang/functional-tests/tests/validator_set/rotate_and_reconfigure.move

```diff
@@ -45,7 +46,7 @@ script {
         // alice rotates bob's public key
         ValidatorConfig::set_config(account, {{bob}},
```

Comment:
Would you adding some tests that call `set_config` with a nonempty vector containing invalid network addresses?